cloud/ec2.go

```diff
@@ -44,6 +44,9 @@ type EC2ProviderSettings struct {
 	// SecurityGroup is the security group name in EC2 classic and the security group ID in a VPC.
 	SecurityGroup string `mapstructure:"security_group" json:"security_group,omitempty" bson:"security_group,omitempty"`
 
+	// SecurityGroupIDs is a list of security group IDs.
+	SecurityGroupIDs []string `mapstructure:"security_group_ids" json:"security_group_ids,omitempty" bson:"security_group_ids,omitempty"`
```

Comment:
Is it possible to just have a single SecurityGroups slice that can sometimes contain a single group in EC2 classic?

Reply:
It's possible but tricky, because the field already exists in the DB and in the providers. Although a little awkward, I think this more easily lets us support lists. I think we can refactor at some point if we want.

Reply:
I think it would be better to make that change and migrate the data though. Having both fields is really awkard

Reply:
What if we keep this like this for now, and make a ticket for the refactor/migration in the project?

Reply:
sgtm